resources/styles/components/learning-guide.less

```diff
@@ -0,0 +1,198 @@
+@lg-width: 1440px;
+@lg-height: 442px;
+
+.learning-guide{
+  width:100%;
+  background-repeat:repeat;
+  position:relative;
+  .x-axis-bg-repeat{
+    // global body padding-left/right prevents line from reaching ends at the moment...body classes with component name?
+    position:absolute;
+    bottom:0;
+    width:100%;
+    height:41px;
+    background:@tutor-lg-green;
+    z-index:-1;
+  }
+  .learning-guide-chart {
+    width:@lg-width;
+    height:@lg-height;
+    margin:0 auto;
+    svg {
+      width: @lg-width;
+      height:@lg-height;
+      .main-title{
+        font-size: .13em; 
+        font-family: Georgia;
+        font-weight:bold;
+        fill:#333333;
+        .show-course{
+          .noselect;
+          cursor:pointer;
+          fill: #1D9FDC;
+          font-size:.8em;
+        }
+      }
+      .circles {
+        .ace{
+          fill: @tutor-tertiary;
+        }
+        .cruising{
+          fill: @tutor-lg-green;
+        }
+        .too-low{
+          fill: @tutor-quaternary;
+        }
+        .grounded{
+          fill: @tutor-neutral-darker;
+        }
+        text {
+          // the unscaled resolution of the svg is only 100px wide,
+          // so 2px is 2% of the width of the chart, i.e. fairly large
+          font-size: 0.119em;
+          fill: white;
+        }
+        circle { stroke:white;stroke-width:.25; }
+        g {
+          cursor: pointer;
+          // &:hover { opacity: 0.8; }
+        }
+      }
+      .grid-lines {
+        line {
+          stroke: lightGrey;
+          stroke-width: 0.1;
+          // stroke-dasharray: 1 2;
+        }
+      }
+      .plot-lines {
+        line {
+          stroke: #FFF;
+          stroke-width: 0.4;
+          stroke-dasharray: 1.2 0.5;
+        }
+      }
+      .background-gradient {
+        // despite the attribute being named "stop-color" this sets the starting color
+        .start { stop-color: #c1d8e5; }
```

Comment:
Can/should these colors be defined elsewhere? (like at the top of the file, since they are not theme-specific)